### ISSUE TITLE
Python Version? requirements.txt

### ISSUE THREAD

**Issue by velotoxicsounds** [dfaulken/iRacing-team-balance-bot#3]
Hello,

which Python Version to use?
Please add requierements.txt

**Comment by dfaulken** [comment]
Thanks for the suggestion @velotoxicsounds. I'm new to Python development. Please let me know if I can add anything else that will make working with it easier. I'll be adding more to the README soon (#4).

**Comment by velotoxicsounds** [comment]
No problem. Has no master fallen from heaven ^^.

Via the requierements.txt you enter your Python modules which you use via "pip install". An example of requierements.txt I have attached to you in the appendix. This also has the advantage if other programmers clone your GIT, which they use via the VIRTUAL_ENV directly install all the required packages. For you, it has the advantage that you have less problems, as you also predefine your used versions of the PIP packages. Not that one user uses setuptools version 1.1.0 and another user uses version 2.0.0.

I've made a fork to your bot. As soon as I have everything I need to get the bot up and running I will be able to help you with "Git-Pushs". On my profile you will also find some Python projects around the topic IRacing, WEB-DEV, Discord Bots. Maybe this is also a small "starter" for you. :)

:) a great collaboration

Greetings
velotoxic
[requirements.txt](https://github.com/dfaulken/iRacing-team-balance-bot/files/5877053/requirements.txt)
[requirements.txt](https://github.com/dfaulken/iRacing-team-balance-bot/files/5877054/requirements.txt)
[requirements.txt](https://github.com/dfaulken/iRacing-team-balance-bot/files/5877055/requirements.txt)
[requirements-test.txt](https://github.com/dfaulken/iRacing-team-balance-bot/files/5877056/requirements-test.txt)
